app/assets/stylesheets/earthworks.css

```diff
@@ -285,3 +285,14 @@ label.toggle-bookmark {
   --bl-facet-active-item-color: rgb(var(--stanford-palo-alto-rgb));
   border-color: rgb(var(--stanford-palo-alto-rgb));
 }
+
+/* Override mobile view for facets section in Blacklight */
+@media (max-width: 991.98px) {
+  .facets-toggleable-md {
+    border: none;
+    padding-top: 0px;
```

Comment:
I would change this to 
.facets {
  --bl-facets-smallish-border: none;
  --bl-facets-smallish-padding: 0;
}

And I think you can get rid of the @media

Reply:
I changed the above, but couldn't figure out how to get facets-heading to change in a similar fashion (I tried --bl-facets-smallish-margin-bottom but the .facets-heading margin-bottom value of 0.5 rem kept overriding that)